invokeai/backend/config/model_install_backend.py

```diff
@@ -149,10 +149,8 @@ def yes_or_no(prompt: str, default_yes=True):
 def get_root(root: str = None) -> str:
     if root:
         return root
-    elif os.environ.get("INVOKEAI_ROOT"):
-        return os.environ.get("INVOKEAI_ROOT")
     else:
-        return config.root
+        return config.root_dir
```

Comment:
`config.root_dir` appears to be a `Path`, but the function typing indicates it always returns a string.